Packages/Book/Sources/Book/Book.swift

```diff
@@ -0,0 +1,49 @@
+import Foundation
+
+public struct Book: Identifiable {
+    /// Title of the book, mandatory field
+    public let title: String
+
+    /// Name of the author, mandatory
+    public let author: String
+
+    /// Release date, optional, should be filled automatically when book is imported
+    public let released: Date?
+
+    /// Genre of the book, optional, should be filled automatically when book is imported
+    public let genre: String?
+
+    /// URL of the service from where/if the book was imported
+    public let url: URL?
+
+    /// A generated ID for display purposes only.
+    ///
+    /// The ID is generated from author and title.
+    /// When user manually adds a book, typing in the ISBN would be cumbersome.
+    public let id: String
+
+    init(
+        title: String,
+        author: String,
+        released: Date? = nil,
+        genre: String? = nil,
+        url: URL? = nil
+    ) {
+        self.title = title
+        self.author = author
+        self.released = released
+        self.genre = genre
+        self.url = url
+        id = "\(author):\(title)"
```

Comment:
I would add release date too if available. Different editions are in theory different books

Reply:
Although that's true I don't see myself having the same book twice in different releases. 
Even if I would, I'd like to have quotes/notes for a single entity in the app. 
But I see your point, I'll flag this and I'll come back to it when building the actual feature around it.